tailnet/test/integration/integration.go

```diff
@@ -111,13 +111,23 @@ type SimpleServerOptions struct {
 
 var _ ServerStarter = SimpleServerOptions{}
 
+type connManager struct {
```

Comment:
Since you're using a type for this instead of a pair of variables, you should just add methods to this type for adding and removing a connection. You could have a single ad method that returns a function to remove.

Reply:
And another method to close all